cubics-obc-test/Components/DeployablesInterface/DeployablesInterface.cpp

```diff
@@ -123,16 +123,18 @@ namespace Components {
         if (FD_ISSET(client_fd, &set)) {
             memset(buffer, 0, sizeof(buffer));
             read(client_fd, buffer, sizeof(buffer));
-            fprintf(stderr, "From server: %s\n", buffer);
-
+            // fprintf(stderr, "From server: %s\n", buffer);
+            this->log_ACTIVITY_LO_receivedDeployableData(buffer);
            this->deployableDataIncoming_out(0, buffer);
+           
         }
     }
   }
 
   void DeployablesInterface :: closeSocket(){
     close(client_fd);
-    fprintf(stderr, "Socket closed \n");
+    // fprintf(stderr, "Socket closed \n");
```

Comment:
Hopefully all the commented out printfs are only there until you verify that logging is working?